File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/Internal/Convert/Conversion.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Internal.Convert;

namespace Db4objects.Db4o.Internal.Convert
{
	/// <exclude></exclude>
	public abstract class Conversion
	{
		/// <param name="stage"></param>
		public virtual void Convert(ConversionStage.ClassCollectionAvailableStage stage)
		{
		}

		/// <param name="stage"></param>
		public virtual void Convert(ConversionStage.SystemUpStage stage)
		{
		}
	}
}

File: db4oj/core/src/com/db4o/internal/convert/Conversion.java
/* Copyright (C) 2004 - 2006  Versant Inc.  http://www.db4o.com */

package com.db4o.internal.convert;

import com.db4o.internal.convert.ConversionStage.*;

/**
 * @exclude
 */
public abstract class Conversion {
    
    /** @param stage */
    public void convert(ClassCollectionAvailableStage stage){
    }
    
    /** @param stage */
	public void convert(SystemUpStage stage){
    }
}
